cocos/profiler/profiler.ts

```diff
@@ -173,6 +176,7 @@ export class Profiler {
             this._showFPS = true;
             this._canvasDone = true;
             this._statsDone = true;
+            legacyCC.game.config.showFPS = true;
```

Comment:
What's the purpose of this ?

Reply:
User may call 'cc.profiler.showStats' or 'cc.profiler.hideStats' to change showing state, after the call, redisplaying the screen will cause the debug panel to return to its previous state if not sync the configuration modification.

This is actually just to sync the configuration modification in debug.setDisplayStats 
![image](https://user-images.githubusercontent.com/23555101/119923792-f3c13780-bfa4-11eb-82b2-ee8adf556ff2.png)